.github/workflows/create-jira-issues-for-community-activities.yml

```diff
@@ -7,14 +7,18 @@ on:
       JIRA_BASE_URL:
         required: true
         description: >
-          Configure as explained under https://github.com/marketplace/actions/jira-login#enviroment-variables. Note that
-          it must NOT end with a slash.
+          Configure as explained under
+          https://github.com/Lombiq/GitHub-Actions/blob/dev/Docs/Workflows/Productivity/CreateJiraIssuesForCommunityActivities.md.
```

Comment:
The parameters need descriptions, and hopefully, there is tooling that makes this easier to consume. Having these as a comment would remove that feature from the workflow's interface. (And even if you look at the code as a newcomer for info about a given parameter, it would make it harder to discover.)

Reply:
I understand the intent and partly agree, although there's no tooling that would rely on having a description, but OK.